<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNDNMovementTester</class>
 <widget class="QDialog" name="DialogNDNMovementTester">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NDN transaction tester</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabStockMovement">
      <attribute name="title">
       <string>Stock document</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="referenceLabel">
           <property name="text">
            <string>Reference</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="referenceLineEdit">
           <property name="maxLength">
            <number>50</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="stornoReferenceLabel">
           <property name="text">
            <string>Storno reference</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="stornoReferenceLineEdit">
           <property name="maxLength">
            <number>30</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="movementTypeLabel">
           <property name="text">
            <string>Movement type</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="movementTypeComboBox"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="reasoncodeLabel">
           <property name="text">
            <string>Reasoncode</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="reasoncodeComboBox"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="issuedOnLabel">
           <property name="text">
            <string>Issued on</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDateTimeEdit" name="issuedOnDateTimeEdit"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="documentDateLabel">
           <property name="text">
            <string>Document date</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QDateTimeEdit" name="documentDateDateTimeEdit"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="commentLabel">
           <property name="text">
            <string>Comment</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="commentLineEdit"/>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="reasonLineEdit"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="reasonLabel">
           <property name="text">
            <string>Reason</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButtonSubmitStockDocument">
         <property name="text">
          <string>Submit stock document</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSales">
      <attribute name="title">
       <string>Sales document</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QFormLayout" name="formLayout_3">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="cashRegisterIDLabel">
           <property name="text">
            <string>Cash register ID:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="cashRegisterIDLineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="referenceLabel_2">
           <property name="text">
            <string>Reference</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="salesReferenceLineEdit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="stornoReferenceLabel_2">
           <property name="text">
            <string>Storno reference</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="salesStornoReferenceLineEdit"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="movementTypeLabel_2">
           <property name="text">
            <string>Movement type</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="salesMovementTypeLineEdit"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="issuedOnLabel_2">
           <property name="text">
            <string>Issued on</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDateTimeEdit" name="salesIssuedOnDateTimeEdit"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="documentDateLabel_2">
           <property name="text">
            <string>Document date</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QDateTimeEdit" name="salesDocumentDateDateTimeEdit"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="dueDateLabel">
           <property name="text">
            <string>Due date</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QDateTimeEdit" name="dueDateDateTimeEdit"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButtonSubmitSalesDocument">
         <property name="text">
          <string>Submit sales document</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabOpeningStock">
      <attribute name="title">
       <string>Opening stock</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <layout class="QFormLayout" name="formLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="openingStockDateLabel">
           <property name="text">
            <string>Opening stock date</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDateTimeEdit" name="openingStockDateDateTimeEdit"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButtonOpeningStock">
         <property name="text">
          <string>Submit opening stock</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QGroupBox" name="groupBoxItems">
     <property name="title">
      <string>Items</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QTableView" name="tableViewItems">
        <property name="minimumSize">
         <size>
          <width>480</width>
          <height>0</height>
         </size>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButtonAddItem">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButtonRemoveItem">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBoxPartner">
     <property name="title">
      <string>Partner</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="localPartnerCodeLabel">
          <property name="text">
           <string>Local partner code:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="localPartnerCodeLineEdit">
          <property name="maxLength">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="nameLabel">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="nameLineEdit">
          <property name="maxLength">
           <number>200</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="cityLabel">
          <property name="text">
           <string>City</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="cityLineEdit">
          <property name="maxLength">
           <number>50</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="postcodeLabel">
          <property name="text">
           <string>Postcode</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="postcodeLineEdit">
          <property name="maxLength">
           <number>4</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="streetLabel">
          <property name="text">
           <string>Street</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="streetLineEdit">
          <property name="maxLength">
           <number>50</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogNDNMovementTester</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogNDNMovementTester</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
